lucupy/minimodel/target.py

```diff
@@ -71,6 +90,14 @@ class SiderealTarget(Target):
     Epoch must be the decimal year.
 
     NOTE: The proper motion adjusted coordinates can be found in the TargetInfo in coord.
+
+    Attributes:
+        ra (float): Right Ascension
+        dec (float): Declination
+        pm_ra (float): ???
+        pm_dec (float): ??
+        epoch (float): ??
```

Comment:
These are:
* Proper motion of the right ascension component
* Proper motion of the declination component
* The epoch in which the ra / dec were measured

Reply:
Thanks! I forgot what those meant so I was hoping you could fill the blanks.